Libraries/Alert/AlertIOS.js

```diff
@@ -244,6 +249,7 @@ class AlertIOS {
       defaultValue,
       cancelButtonKey,
       destructiveButtonKey,
+      keyboardType
```

Comment:
Add a comma at the end of this line. We follow this convention to ensure we don't need to update the preceding line when adding parameters.